pkg/v1/tkg/managementcomponents/helper.go

```diff
@@ -21,18 +23,18 @@ const (
 )
 
 // GetTKGPackageConfigValuesFileFromUserConfig returns values file from user configuration
-func GetTKGPackageConfigValuesFileFromUserConfig(managementPackageVersion string, userProviderConfigValues map[string]interface{}) (string, error) {
+func GetTKGPackageConfigValuesFileFromUserConfig(managementPackageVersion string, userProviderConfigValues map[string]interface{}, tkgBomConfig *tkgconfigbom.BOMConfiguration) (string, error) {
```

Comment:
We're adding an additional input to this function. Are there corresponding unit tests that need to get updated / added?

Reply:
Thanks for catching this. I added unit for this function explicitly.